tests/Cms/BannerServiceTest.php

```diff
@@ -104,6 +105,25 @@ public function testGetNonExistentCategoryBanners()
         $this->assertEmpty($banners);
     }
 
+    public function testGetBannersWithAltTags()
+    {
+        $cmsService = $this->buildBannerService();
+
+        $banners1 = $cmsService->getHomepageBanners();
+        $this->assertSame(2, count($banners1));
```

Comment:
pareil pour les autres appels dynamiques alors que c'est statique
```suggestion
        static::assertSame(2, count($banners1));
```